### DISCUSSION TITLE
[question] how to enable ibus compatible mode in fcitx5?

### DISCUSSION THREAD

**Discussion by Freed-Wu** [fcitx/fcitx5#628] [Q&A] [answered]
I do <https://wiki.archlinux.org/title/Fcitx5#Integration>, but <https://github.com/kovidgoyal/kitty/issues/5566#issuecomment-1290773337> tell me I must enable ibus compatible mode in fcitx5. I cannot find any content about ibus compatible mode.

Thanks!

**Comment by wengxt** [comment, marked as answer]
There’s nothing you need to do on fcitx5 side, it’s built in and should be enabled automatically.

but you’ll need to set 
GLFW_IM_MODULE=ibus for kitty if you’re under x11.

If you’re running kitty under wayland (not under xwayland), you need to use a compositor that support wayland input method (text-input-v3). This includes gnome shell, kwin, or sway.

**Reply by Freed-Wu** [reply]
I have tested it work for some builtin language but cannot work for fcitx-libpinyin. Is the bug related to libpinyin not fcitx?

**Reply by wengxt** [reply]
are you using fcitx5 or fcitx?
there’s no fcitx5-libpinyin

**Reply by Freed-Wu** [reply]
Oh, I am using fcitx5 and fcitx-libpinyin. It looks like fcitx5 can support fcitx-libpinyin, right?

**Reply by wengxt** [reply]
fcitx5 never supports libpinyin. Are you talking about something else? (fcitx5-zhuyin maybe?)

**Reply by Freed-Wu** [reply]
I have changed to fcitx5-rime and it works well. Thanks!
